crates/transport_tx5/src/test.rs

```diff
@@ -525,9 +536,157 @@ async fn preflight_send_recv() {
     .unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
I know it's the style of the existing tests here, but I invite you to break that pattern and use more descriptive identifiers for variables. One letter names always require me to look twice what something is.